.changeset/beige-rats-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-components': minor
+---
+
+feat: make possible to project to upgrade virtualized list
```

Comment:
?

Reply:
oups, this one was a missing file from previous PR 🤦 
Thanks to point it out.